src/main/java/com/questhelper/helpers/quests/theheartofdarkness/TheHeartOfDarkness.java

```diff
@@ -0,0 +1,1028 @@
+                "the area.");
+        inspectFireMarkings = new ObjectStep(this, NullObjectID.NULL_55361, new WorldPoint(1613, 9642, 0), "Inspect the fire markings in the north-west of " +
+                "the area.");
+
+        activateFirstStatue = new ObjectStep(this, NullObjectID.NULL_54459, new WorldPoint(1650, 9642, 0), "Activate the first statue in the order.");
+        activateFirstStatue.addAlternateObjects(NullObjectID.NULL_54465, NullObjectID.NULL_54471, NullObjectID.NULL_54477);
+        activateSecondStatue = new ObjectStep(this, NullObjectID.NULL_54459, new WorldPoint(1650, 9642, 0), "Activate the second statue in the order.");
+        activateSecondStatue.addAlternateObjects(NullObjectID.NULL_54465, NullObjectID.NULL_54471, NullObjectID.NULL_54477);
+        activateThirdStatue = new ObjectStep(this, NullObjectID.NULL_54459, new WorldPoint(1650, 9642, 0), "Activate the third statue in the order.");
+        activateThirdStatue.addAlternateObjects(NullObjectID.NULL_54465, NullObjectID.NULL_54471, NullObjectID.NULL_54477);
+        activateFourthStatue = new ObjectStep(this, NullObjectID.NULL_54459, new WorldPoint(1650, 9642, 0), "Activate the fourth statue in the order.");
+        activateFourthStatue.addAlternateObjects(NullObjectID.NULL_54465, NullObjectID.NULL_54471, NullObjectID.NULL_54477);
+        statueActivateSteps[0] = activateFirstStatue;
+        statueActivateSteps[1] = activateSecondStatue;
+        statueActivateSteps[2] = activateThirdStatue;
+        statueActivateSteps[3] = activateFourthStatue;
+
+        enterFinalBossRoom = new ObjectStep(this, NullObjectID.NULL_55355, new WorldPoint(1601, 9631, 0), "Enter the door to the west, ready for the boss.");
+
+        defeatAmoxliatl = new NpcStep(this, NpcID.AMOXLIATL, new WorldPoint(1365, 4510,  0), "Defeat Amoxliatl. See the sidebar for more details.");
+        ((NpcStep) defeatAmoxliatl).addAlternateNpcs(NpcID.AMOXLIATL_13686, NpcID.AMOXLIATL_13687, NpcID.AMOXLIATL_13689);
+
+        defeatAmoxliatlSidebar = new DetailedQuestStep(this, "Defeat Amoxliatl. Use Protect from Magic.");
+        defeatAmoxliatlSidebar.addText("Ice spikes will appear on the floor, move off the tile to avoid them.");
+        defeatAmoxliatlSidebar.addText("She may disable your prayers and shoot an icicle at you. Simply turn back on Protect from Magic.");
+        defeatAmoxliatlSidebar.addText("Ice blocks may appear. Use melee to destroy them all before they explode and heal the boss.");
+        defeatAmoxliatlSidebar.addSubSteps(defeatAmoxliatl);
+
+        talkToServius = new NpcStep(this, NpcID.SERVIUS_TEOKAN_OF_RALOS_13694, new WorldPoint(1681, 3168, 0), "Talk to Servius, Teokan of Ralos in the palace" +
+                " in Civitas illa Fortis to complete the quest.");
+        talkToServius.addTeleport(civitasIllaFortisTeleport);
```

Comment:
The teleport should be added to the quest recommended items too - it was a surprise to me when I got to this section